### DISCUSSION TITLE
[react-router-dom] React Router v6 still shows type definitions for v5

### DISCUSSION THREAD

**Discussion by TheTrio** [DefinitelyTyped/DefinitelyTyped#59342] [Issues with a @types package] [answered]
I don't know if this is a problem with my setup but despite upgrading to react router v6 (which has its own type definitions), VS Code continues to believe that the version of react router is 5. In the @types folder I have a directory named `react-router-dom`

![image](https://user-images.githubusercontent.com/10794178/159068130-c5bc4c45-67f9-4430-85a9-dccd2ba7f476.png)

Is there a way to fix this so that I can get the proper intellisense?

[a comment by an automated account is not shown]

**Comment by eps1lon** [comment]
For v6 you should not install `@types/react-router-dom` since `react-router-dom` ships their own types. So if you're using v6 you can remove `@types/react-router-dom`

**Comment by TheTrio** [comment, marked as answer]
I didn't install the types manually.

Edit: I think this might be a vs code thing. Looks like it downloads [the types automatically](https://code.visualstudio.com/Docs/languages/javascript#_intellisense-is-not-working-for-external-libraries)

**Reply by eps1lon** [reply]
Check `yarn why @types/react-router-dom` or `npm ls @types/react-router-dom`. VSCode does not install the types if the package in question ships their own.

But it's always possible that you have an older version of `react-router-dom` in your package tree somewhere.

In any case, without a repro it's hard to know what's wrong.

**Reply by TheTrio** [reply]
It doesn't look like its installed 
![image](https://user-images.githubusercontent.com/10794178/159128132-42959354-f4f1-40a3-b87f-0e22ef384472.png)

For what its worth, I was able to solve this by ignoring the `react-router-dom` package inside of my tsconfig

```
{
  "typeAcquisition": {
      "enable": true,
      "exclude": ["react-router-dom"]
  }

}
```
